lib/broadside/configuration.rb

```diff
@@ -26,6 +26,8 @@ def initialize
       @logger.datetime_format = '%Y-%m-%d_%H:%M:%S'
       @timeout = 600
       @type = 'ecs'
+      @ssh = {}
+      @targets = {}
```

Comment:
i don't think you need to instantiate this here, especially given that on my branch i am `validate`ing that the targets exist and that would make it weird